app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -97,68 +120,138 @@ const MapImpl: React.FunctionComponent = () => {
       case MapFeatureState.LOADED: {
         /*
         Features in this state have been queried, created and pushed into the
-        features state list that is located on the store. The next step is to
-        add the data of the features to the map using the addDataToMap action which is reduced by
-        keplerGL reducer.. The geometry of the feature is used
-        to display its location and shape on the map.
+        features state list that is located on the store. They have also been pushed into the appropriate place of the 
```

Comment:
Now that I look at this, could you have accomplished the same thing with another MapFeatureState?
AWAITING_ADD?